util/dvsim/dvsim.py

```diff
@@ -142,11 +142,12 @@ def main():
                         metavar="vcs|xcelium|ascentlint|veriblelint|dc|...",
                         help="Override the tool that is set in hjson file")
 
-    parser.add_argument("-select_cfgs",
-                        nargs="*",
-                        default=[],
-                        metavar="cfg1, cfg2, cfg3, ...",
-                        help="""Specifies which cfg(s) of the master cfg shall be processed.
+    parser.add_argument(
```

Comment:
Its a shame that these unintended changes are showing up. Would you please resubmit without it?

Reply:
That is done due to yapf. Shouldn't we use yapf for python script formatter?

Reply:
yapf is super flaky and produces a different change every time.